### PR TITLE
Add option to hide ShellshapeIndicator

### LINKED ISSUE #155: Add option to hide ShellshapeIndicator
I think it would be a good idea to add option to hide ShellshapeIndicator. Shellshape is fully usable even without using the indicator, so some users may find it useful to hide the indicator to conserve space in the status area.

### PR DESCRIPTION
**Pull request by jdoe0**
Fixes #155.

### CODE REVIEW

File: src/gjs/prefs.ts [hunk 1]
```diff
@@ -34,6 +34,35 @@ function buildPrefsWidget() {
...
+
+		var checkbutton = new Gtk.CheckButton({ label: _("Show indicator in status panel") });
+
+		hbox.pack_end(checkbutton, true, true, 0);
+		vbox.add(hbox);
+
+		var pref = config.SHOW_INDICATOR;
+		checkbutton.set_active(pref.get());
+		checkbutton.connect('toggled', function(sw) {
+			var oldval = pref.get();
```
Comment (gfxmonk) [inline src/gjs/prefs.ts]: `oldval` is not used anywhere

**Comment by gfxmonk** [comment]
Awesome - aside from the one nitpick above, this looks good to go :)

**Comment by jdoe0** [comment]
There are many unused `oldval`s in existing code as well. Should I remove them all?

**Comment by gfxmonk** [comment]
Hah, how embarrassing! No worries, I've removed them all now. Thanks for the patch :)

**Pull request merged**